shared/modules/pocket_bus_module.go

```diff
@@ -4,14 +4,14 @@ import (
 	"github.com/pokt-network/pocket/shared/types"
 )
 
-type EventsChannel chan types.Event
+type EventsChannel chan types.PocketEvent
 
 type Bus interface {
 	// TODO: Do we want to implement `Module` here as well?
 
 	// Bus Events
-	PublishEventToBus(e *types.Event)
-	GetBusEvent() *types.Event
+	PublishEventToBus(e *types.PocketEvent)
+	GetBusEvent() *types.PocketEvent
```

Comment:
(_Just a comment_): I remember a short conversation about idiomatic go. I noticed also that in the generated protobuff golang code, they do not seem to respect the idiomatic way of naming getters. Overall, it's simpler to read `GetBusEvent` than `BusEvent`. Still, I want to hear other opinions, particularly yours.

Reply:
I have no strong opinions either way. I think either approach enables us to do what we need.